templates/Makefile.template

```diff
@@ -347,6 +347,31 @@ HOST_LDLIBS = $(LDLIBS)
 
 HAS_PKG_CONFIG = $(shell command -v pkg-config >/dev/null 2>&1 && echo true || echo false)
 
+PC_TEMPLATE = prefix=$(prefix)\n\
+exec_prefix=${'\$${prefix}'}\n\
+includedir=${'\$${prefix}'}/include\n\
+libdir=${'\$${exec_prefix}'}/lib\n\
+\n\
+Name: $(PC_NAME)\n\
+Description: $(PC_DESCRIPTION)\n\
+Version: $(VERSION)\n\
+Cflags: -I${'\$${includedir}'} $(PC_CFLAGS)\n\
+Requires.private: $(PC_REQUIRES_PRIVATE)\n\
+Libs: -L${'\$${libdir}'}\n\
+Libs.private: $(PC_LIBS_PRIVATE)
+
+# gpr .pc file
+PC_NAME = Google Public Runtime
+PC_DESCRIPTION = Google Public Runtime
+PC_CFLAGS =
+PC_REQUIRES_PRIVATE =
```

Comment:
I think P was for portable.

On Wed, Jul 1, 2015 at 10:23 AM Nicolas Noble <notifications@github.com>
wrote:

> In templates/Makefile.template
> <https://github.com/grpc/grpc/pull/2268#discussion_r33701706>:
>
> > +includedir=${'\$${prefix}'}/include\n\
> > +libdir=${'\$${exec_prefix}'}/lib\n\
> > +\n\
> > +Name: $(PC_NAME)\n\
> > +Description: $(PC_DESCRIPTION)\n\
> > +Version: $(VERSION)\n\
> > +Cflags: -I${'\$${includedir}'} $(PC_CFLAGS)\n\
> > +Requires.private: $(PC_REQUIRES_PRIVATE)\n\
> > +Libs: -L${'\$${libdir}'}\n\
> > +Libs.private: $(PC_LIBS_PRIVATE)
> > +
> > +# gpr .pc file
> > +PC_NAME = Google Public Runtime
> > +PC_DESCRIPTION = Google Public Runtime
> > +PC_CFLAGS =
> > +PC_REQUIRES_PRIVATE =
>
> Wasn't this supposed to read -pthreads and -lpthreads ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/2268/files#r33701706>.
>